src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForManagedIdentityParameterBuilder.cs

```diff
@@ -59,6 +59,25 @@ public AcquireTokenForManagedIdentityParameterBuilder WithForceRefresh(bool forc
             return this;
         }
 
+        /// <summary>
+        /// Sets claims in the query. Use when the AAD admin has enabled conditional access evaluation. Acquiring the token normally will result in a
```

Comment:
Proposed comment

`Adds a claims challenge to the token request. The SDK will bypass the token cache when a claims challenge is specified. 

Remarks: 

this API is not always available on the client, in which case MSAL will not inject the claims. Apps wanting to monitor availability may use <see ...>

A claims challenge typically arises when calling the protected downstream API, for example when the tenant administrator wants to revokes credentials. Apps are required to look for a 401 Unauthorized response from the protected api and to parse the WWW-Authenticate response header in order to extract the claims. For more details see https://learn.microsoft.com/entra/identity/conditional-access/concept-continuous-access-evaluation